fabtests/common/shared.c

```diff
@@ -2850,6 +2851,12 @@ void ft_parse_addr_opts(int op, char *optarg, struct ft_opts *opts)
 		else
 			opts->oob_port = default_oob_port;
 		break;
+	case 'F':
+		if (strcmp(optarg, "FI_FORMAT_UNSPEC") == 0)
+			opts->address_format = FI_FORMAT_UNSPEC;
+		else if (strcmp(optarg, "FI_SOCKADDR_IB") == 0)
+			opts->address_format = FI_SOCKADDR_IB;
```

Comment:
We can drop the unspec check, as that's the default.  Please use a case insensitive comparison.  We should add the other common options here as well, since the user does not know that other values get ignored.

Reply:
Done. I now use `strncasecmp` and I have added common address formats: `fi_sockaddr`, `fi_sockaddr_in`, `fi_sockaddr_in6` and `fi_sockaddr_ib`